contracts/src/arbitration/IArbitrator.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8;
+
+import "./IArbitrable.sol";
+
+/**
+ * @title Arbitrator
+ * Arbitrator interface for CourtV2.
+ * This interface for the most part follows the ERC-792 standard but also allows the appeal crowdfunding on arbitrator's side.
```

Comment:
"follows ~~for the most part~~ the ERC-792 standard" 

I suggest that we either:
1. We amend the ERC-792 specification so that this interface complies with it - but the specs must deal with v1 compatibility. The amendment can be made outside [the ERC-1 process](https://eips.ethereum.org/EIPS/eip-1#eip-process) because the status is "pre-draft" and not formally tracked.
2. Or we start a new ERC spec which leaves kleros-v1 compatibility untouched at the expense of having to rebuild legitimacy for this new ERC from scratch.

Reply:
Personally,I was thinking that it'll be a new ERC